frontend/sphinx/_templates/analytics.html

```diff
@@ -27,3 +27,15 @@
     // End LinkedIn Tracking Tags
   }
 </script>
+<script type="text/javascript">
+    var __raconfig = __raconfig || {};
+    __raconfig.uid = '6267eb47994c2';
+    __raconfig.action = 'track';
+    (function () {
+        var ra = document.createElement('script');
+        ra.type = 'text/javascript';
+        ra.src = 'https://ruler.nyltx.com/lib/1.0/ra-bootstrap.min.js';
+        var s = document.getElementsByTagName('script')[0];
+        s.parentNode.insertBefore(ra, s);
```

Comment:
Note that, by putting the code in this separate `<script>` chunk, the analytics will include activities from AdaCore staff. There's some code above to prevent that, see line 4:

```javascript
if(!Cookies.get('AdaCore_staff')) {
```

Reply:
Right, we could add the code in the upper script.